### PR TITLE
Rolling and tf idf

### PR DESCRIPTION
**Pull request by mqharris**
applies rolling sums with window 2 if you pass rolling=True to fit_transform()

rolling=True slows down the code a fair bit, so if we care about speed it could be moved and vectorized

I was getting a random order of columns each time so I sorted the columns in one step of the processing

### CODE REVIEW

File: process/data_processor.py [hunk 1]
```diff
@@ -100,30 +101,38 @@ def fit_transform_subblocks(self, X_seq, term_weighting=None):
             # put into dataframe to add nas to missing events
             # divide by 20 as original operation multiplied by 20
             block_df = pd.DataFrame(block_counts, columns=self.events) / 20
+            block_df = block_df.reindex(sorted(block_df.columns), axis=1)
             block_df = block_df.fillna(0)
+            if self.rolling:
```
Comment (mqharris) [inline process/data_processor.py:106]: I started by putting it here since we were already touching the data frames

but if care about optimizing it we could move some stuff around

File: process/data_processor.py [hunk 2]
```diff
@@ -100,30 +101,38 @@ def fit_transform_subblocks(self, X_seq, term_weighting=None):
             # put into dataframe to add nas to missing events
             # divide by 20 as original operation multiplied by 20
             block_df = pd.DataFrame(block_counts, columns=self.events) / 20
+            block_df = block_df.reindex(sorted(block_df.columns), axis=1)
```
Comment (mqharris) [inline process/data_processor.py:104]: sort the columns

I was getting random column order each time i ran it
which lead to the training set looking different from the test set

**Review by WraySmith — APPROVED** [review]
Looks good, just need to abstract run code into an interactive file and create the README in the folder (similar but should be shorter than the parse folder). Lets get this branch merged in and then the folder clean-up can be done in another branch while the NN code is started.
